pipeline_steps/common.groovy

```diff
@@ -22,10 +22,10 @@ def install_ansible(){
     source .venv/bin/activate
 
     # These pip commands cannot be combined into one.
-    pip install -U six packaging appdirs
-    pip install -U setuptools
-    pip install 'pip==9.0.1'
+    pip install --index-url https://pypi.python.org/simple 'pip==9.0.1 setuptools==33.1.1'
```

Comment:
Actually, I was wrong. The issue is the quotes. This works:

pip install --index-url https://pypi.python.org/simple pip==9.0.1 setuptools==33.1.1
pip install --isolated -c rpc-gating/constraints.txt six packaging appdirs